### PR TITLE
Fix Text Modal Issues

### PR DESCRIPTION
**Pull request by udhadpa**
Fixed text modal #4 issues.
All tests were failing due to the change in the html file.
Id's were changed.

@umahaea

### CODE REVIEW

**Comment by StommePoes** [comment]
looks good

**Comment by udhadpa** [comment]
Merging the changes

File: src/main/java/origamiV2/jsfiles/textModal/temp.js [hunk 1]
```diff
@@ -1 +1,9 @@
+function init() {
 
+  // Demo eventing API
+  document.body.dispatchEvent(new CustomEvent('o.InitTextModal', {
+      detail: { elementId: 'app', contentTemplateLarge: true, footerVisible: true, successBtnCallback: function() { console.log('¡¡success button pressed!!') }}
+  }
+  ));
+}
```
Comment (umahaea) [inline src/main/java/origamiV2/jsfiles/textModal/temp.js]: lets keep the temp file always empty.

File: src/test/java/origamiV2Tests/TextModalTest.java [hunk 2]
```diff
@@ -27,11 +28,16 @@
     private final String textModalJSPath = "/home/travis/build/Pearson-Higher-Ed/ux-test-platform/src/main/java/origamiV2/jsfiles/textModal/text-modal.js";
     private final String tempJSFilePath = "/home/travis/build/Pearson-Higher-Ed/ux-test-platform/src/main/java/origamiV2/jsfiles/textModal/temp.js";
     private final String textModalDistJSFilePath = "/home/travis/build/Pearson-Higher-Ed/ux-test-platform/src/main/java/origamiV2/jsfiles/textModal/dist.text-modal.js";
+
+    /*private final String textModalJSPath = "/Users/udhadpa/ux-test-platform/src/main/java/origamiV2/jsfiles/textModal/text-modal.js";
```
Comment (umahaea) [inline src/test/java/origamiV2Tests/TextModalTest.java]: we don't need to have this local references.

File: src/test/java/origamiV2Tests/TextModalTest.java [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 import org.apache.log4j.Logger;
 import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.WebDriverWait;
```
Comment (umahaea) [inline src/test/java/origamiV2Tests/TextModalTest.java]: do we use this anywhere?

**Comment by umahaea** [comment]
sorry for the late comments. please fix them probably in your next PR.

**Comment by udhadpa** [comment]
sure. @umahaea

**Pull request merged**